File: Week 6-DNA+sentimental-cash,mario,readability/sentimental-mario/mario.py
# TODO
from cs50 import get_int

# get user input
while True:
    h = get_int("Height: ")
    if h > 0 and h <= 8:
        break

blank = h - 1  # To print blank spaces before #
for i in range(h):
    print(" " * blank, end="")
    blank -= 1
    print("#" * (i + 1))

File: Week 1-cash+mario/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //x = get user input
    int x;
    do
    {
        x = get_int("Height: ");
    }
    while (x < 1 || x >= 9);

    //assign number of rows acording to user imput
    for (int r = 0; r >= 0 && r < x; r++)
    {
        int c = x;
        while (c > r + 1)
        {
            printf(" ");
            c--;
        }
        for (c = r; c >= 0; c--)
        {
            printf("#");
        }
        printf("\n");
    }
}